README.md

```diff
@@ -34,15 +34,21 @@ The database starts for a long time and the history tools may fall with an error
 nodeos -e -p eosio --config-dir `pwd`environment/ --delete-all-blocks --disable-replay-opts
 ```
 Option `--disable-replay-opts` is needed for `state-history-plugin`
+
+### Database Migrations
+Install tool [dbmate](https://github.com/amacneil/dbmate#installation) and run:
+```
+GO111MODULE=on go get -u github.com/amacneil/dbmate
+dbmate up
```

Comment:
so, I guess that dbmate util should be installed in container with service.
Else we would have to install dbmate to host system, and it's not good way